src/main/java/com/maxdemarzi/shortest/Validators.java

```diff
@@ -22,6 +22,10 @@ public static HashMap getValidQueryInput(String body) throws IOException {
         if (!input.containsKey("center_email")) {
             throw Exceptions.missingCenterEmailParameter;
         }
+        // Make sure it has a bibliography_entries parameter
+        if (!input.containsKey("bibliography_entries")) {
```

Comment:
yeah, i was planning to deploy a new version of the emailservice first, we shouldnt need any down time